src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs

```diff
@@ -421,6 +421,17 @@ internal static DType ClosestParentScopeTypeForSuggestions(CallNode callNode, In
             }
         }
 
+        internal static IEnumerable<KeyValuePair<string, DType>> GetColumnNameIdentifierSuggestions(DType scopeType)
```

Comment:
>GetColumnNameIdentifierSuggestions [](http://example.com/codeflow?start=65&length=34)

where is this used?

Reply:
This will be used in PAClient intellisense